packages/flutter/lib/src/cupertino/app.dart

```diff
@@ -143,6 +187,18 @@ class CupertinoApp extends StatefulWidget {
   /// {@macro flutter.widgets.widgetsApp.navigatorObservers}
   final List<NavigatorObserver> navigatorObservers;
 
+  /// {@macro flutter.widgets.widgetsApp.routeInformationParser}
+  final RouteInformationParser<Object> routeInformationParser;
+
+  /// {@macro flutter.widgets.widgetsApp.routerDelegate}
+  final RouterDelegate<Object> routerDelegate;
+
+  /// {@macro flutter.widgets.widgetsApp.backButtonDispatcher}
+  final BackButtonDispatcher backButtonDispatcher;
+
+  /// {@macro flutter.widgets.widgetsApp.routeInformationProvider}
+  final RouteInformationProvider routeInformationProvider;
+
```

Comment:
trivial nit but please (as per our style guide) put these in the  same order everywhere and ideally an order that has the four delegates either in the order that they get normally called or in the order that you'd want to learn about them in. Right now the code is inconsistent about what order these four names are sometimes put in.